### PR TITLE
featureR: html file has been changed

### PR DESCRIPTION
**Pull request by ramilvl**

### CODE REVIEW

**Review by nadir-pb — COMMENTED** [review]
test

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    <span>Test</span>
+    <span>First Commit</span>
```
Comment (nadir-pb) [inline index.html:9]: extra

Comment (nadir-pb) [reply index.html:9]: test

**Review by nadir-pb — DISMISSED** [review]

**Review by gunayahmadzada — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (nadir-pb) [inline index.html:9]: niye deyismisen?

File: index.html [hunk 2]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
```
Comment (yyusifov) [inline index.html:7]: test

**Review by yyusifov — DISMISSED** [review]

**Review by yyusifov — APPROVED** [review]

**Review by nadir-pb — APPROVED** [review]

**Review by gunayahmadzada — APPROVED** [review]

**Pull request merged**